recipes/pydsd/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "PyDSD" %}
+{% set version = "1.0.3.2" %}
+{% set sha256 = "f0b815e6a1356e56a92e505cf67530cfef69588cceeebf9489f5ad5a5fc036c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
```

Comment:
Does this package not run on windows?

Reply:
Not as of yet. We had issues getting pytmatrix to run on windows and so I've never been able to test it. I plan on making pytmatrix an optional dependency eventually which should allow it to run on windows.